### ISSUE TITLE
Scaling disallowed?

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2239]
https://github.com/harvard-lil/perma/commit/ac9e5e774aa7c42517aeb1c036e25d64542a8565
https://github.com/harvard-lil/perma/commit/34fd00232d3e259dc5ebd490f5f231804a89c507

These appear to be efforts to prevent scaling. Scaling is an important feature for users with low vision. Viewport elements are great, but minimum scale and maximum scale should never be set. Users should always be allowed to select the scale that works for them.

**Comment by rebeccacremona** [comment]
Safari has since changed their behavior to permit zooming at all times:
https://twitter.com/thomasfuchs/status/742531231007559680/photo/1
http://cloudless.studio/articles/39-safari-zoom-gesture-s-comeback-in-ios-10

But, we checked on an android phone, and pinch-to-zoom was not possible.

**Comment by rebeccacremona** [comment]
Bootstrap and html5-boilerplate currently share the same recommendations: https://perma.cc/K4DA-47YN 

`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`

Long discussion showing how difficult this all is at https://github.com/h5bp/html5-boilerplate/issues/1985.

I'm going to try this out on Perma, and we'll do the best we can to test it on a variety of mobile devices. I'm encouraged by preliminary experiments: I think we should be okay..... 🤞
